docs/ovn_k8s.conf.5

```diff
@@ -23,45 +22,66 @@ This represents the conntrack zone used for the conntrack flow rules.
 .PP
 .SH [Logging]
 .TP
-\fBlogfile\fR=/var/log/openvswitch/ovn-k8s-cni-overlay.log
-Logfile name (with path) for ovnkube to write to.
+\fBlogfile\fR=
+When logging to a file, this is the path and file. E.g., /var/log/openvswitch/ovn-k8s-cni-overlay.log
+Default is not to log to a file.
 .TP
-\fBloglevel\fR=0
-Level is the logging verbosity level: 5=debug, 4=info, 3=warn, 2=error, 1=fatal (default: 0).
+\fBloglevel\fR=4
+Level is the logging verbosity level: 5=debug, 4=info, 3=warn, 2=error, 1=fatal (default: 4).
 .SH [CNI]
 .PP
 The following config values are used for the CNI plugin.
 .TP
-\fBcni-conf-dir\fR=/etc/cni/net.d
+\fBconf-dir\fR=/etc/cni/net.d
 Directory that contains the cni plugin.
 .TP
-\fBcni-plugin\fR=ovn-k8s-cni-overlay
+\fBplugin\fR=ovn-k8s-cni-overlay
 Cni plugin name.
 .SH [Kubernetes]
 .PP
-K8S apiserver and authentication details are stored in the following options.
+K8S apiserver and authentication details are declared in the following options.
 .TP
 \fBkubeconfig\fR=
 Absolute path to the kubeconfig file (not required if the apiserver, cacert, and token are given).
 .TP
-\fBcacert\fR=/etc/openvswitch/k8s-ca.crt
-The absolute path to the Kubernetes API CA certificate (not required if kubeconfig is given).
+\fBcacert\fR=
+The absolute path to the Kubernetes API CA certificate, e.g.,
+/etc/openvswitch/k8s-ca.crt (not required if kubeconfig is given).
 .TP
-\fBapiserver\fR=
-URL to the Kubernetes apiserver (default "https://localhost:8443") (not required if kubeconfig is given).
+\fBapiserver\fR=http://localhost:8080
+URL to access the Kubernetes apiserver. The host must be a name, not IP. The kubernetes/openshift
+api serveri, by default, listens on 8443. E.g., https://masterhost:8443"
+(Not required if kubeconfig is given.)
 .TP
 \fBtoken\fR=
-The Kubernetes API authentication token (not required if kubeconfig is given).
+The Kubernetes API authentication token permits the ovn plugin to communicate with the
+api server. On openshift, the token can be generated on the cluster master as follows:
+
+# oc create serviceaccount ovn
+
+# oc adm policy add-cluster-role-to-user cluster-admin -z ovn
+
+# oc adm policy add-scc-to-user anyuid -z ovn
+
+# oc sa get-token ovn
+
+(not required if kubeconfig is given).
 
 .SH [OvnNorth]
 .TP
 \fBaddress\fR=
+This is the url used to access the northbound database server. The scheme may be ssl or tcp.
+When ssl is used, the certs must be provided. The host must be a host IP address, not name.
+The port, by default, is 6441. E.g., ssl://1.2.3.4:6641
 .TP
-\fBclient-privkey\fR=/etc/openvswitch/ovncontroller-privkey.pem
+\fBclient-privkey\fR=
+For example: /etc/openvswitch/ovncontroller-privkey.pem
 .TP
-\fBclient-cert\fR=/etc/openvswitch/ovncontroller-cert.pem
+\fBclient-cert\fR=
+For example: /etc/openvswitch/ovncontroller-cert.pem
 .TP
-\fBclient-cacert\fR=/etc/openvswitch/ovnnb-ca.cert
```

Comment:
From what I see in the code, those still have the default values:
https://github.com/openvswitch/ovn-kubernetes/blob/master/go-controller/pkg/config/config.go#L401-L406
It would be:
```
client-privkey\fR=/etc/openvswitch/ovnnb-privkey.pem
client-cert\fR=/etc/openvswitch/ovnnb-cert.pem
client-cacert\fR=/etc/openvswitch/ovnnb-ca.cert
```

Reply:
You can see the default values in the code. (https://github.com/openvswitch/ovn-kubernetes/blob/master/go-controller/pkg/config/config.go#L408-L414)